kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/filtered-resources/kjars-sources/webservice-project/src/main/resources/travelAgencyWrappedParamsProcess.bpmn2

```diff
@@ -0,0 +1,226 @@
+_flightReq.setStartCity(&quot;Paris&quot;);
+_flightReq.setEndCity(&quot;Berlin&quot;);
+_flightReq.setStartDate(&quot;10-10-2018&quot;);
+_flightReq.setEndDate(&quot;10-11-2018&quot;);
+/*kcontext.setVariable(&quot;requestFlight&quot;, _flightReq);*/
+com.jboss.soap.service.acmedemo.ListAvailablePlanes _listPlanes = new com.jboss.soap.service.acmedemo.ListAvailablePlanes();
+_listPlanes.setIn(_flightReq);
+kcontext.setVariable(&quot;listPlanes&quot;, _listPlanes);
+System.out.println(&quot;Completed setting WS variables for both. moving on now!!.....&quot;);
+/* end WS variables */
+
+System.out.println(&quot;Leaving &quot; + kcontext.getNodeInstance().getNodeName() +&quot; Node&quot;);</bpmn2:script>
+    </bpmn2:scriptTask>
+    <bpmn2:sequenceFlow id="_F62E19E4-6F3B-45FB-8B6B-730D10233A9E" drools:selectable="true" color:background-color="#000000" color:border-color="#000000" color:color="#000000" sourceRef="_553B4C08-C4D4-4337-86A7-274F96A12773" targetRef="_3FD70F6B-AD40-4887-A3BA-B02C5093B685"/>
+    <bpmn2:task id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F" drools:selectable="true" drools:taskName="WebService" color:background-color="#00FFFF" color:border-color="#000000" color:color="#000000" name="Flight Web Service">
+      <bpmn2:extensionElements>
+        <drools:metaData name="elementname">
+          <drools:metaValue><![CDATA[Flight Web Service]]></drools:metaValue>
+        </drools:metaData>
+        <drools:onEntry-script scriptFormat="http://www.java.com/java">
+          <drools:script>System.out.println(&quot;Entering &quot; + kcontext.getNodeInstance().getNodeName() +&quot; Node&quot;);
+</drools:script>
+        </drools:onEntry-script>
+        <drools:onExit-script scriptFormat="http://www.java.com/java">
+          <drools:script>kcontext.setVariable(&quot;ratePerPerson&quot;,listResponse.getReturn().getRatePerPerson().toString());
+System.out.println(&quot;[Flight Web Service Task] YOU will be flying soon..YIPEEE.....your rate: &quot; + listResponse.getReturn().getRatePerPerson()); 
+kcontext.setVariable(&quot;listPlanes&quot;, null);
+kcontext.setVariable(&quot;listResponse&quot;, null);
+System.out.println(&quot;Leaving &quot; + kcontext.getNodeInstance().getNodeName() +&quot; Node&quot;);
+
+</drools:script>
+        </drools:onExit-script>
+      </bpmn2:extensionElements>
+      <bpmn2:incoming>_492148F7-AB4F-4064-A972-AAAAF2B0D5A0</bpmn2:incoming>
+      <bpmn2:outgoing>_E0F59797-6B00-4B4C-9E74-04FBCD671914</bpmn2:outgoing>
+      <bpmn2:ioSpecification id="_GgtfMYtEEeiO7cOOapYI8w">
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_TaskNameInputX" drools:dtype="String" itemSubjectRef="__34658F5F-BF7B-41BB-99ED-AD655FED5A4F_TaskNameInputXItem" name="TaskName"/>
+        <bpmn2:dataInput id="_34658F5F-BF7B-41BB-99ED-AD655FED5A4F_ParameterInputX" drools:dtype="com.jboss.soap.service.acmedemo.FlightRequest" itemSubjectRef="ItemDefinition_1237" name="Parameter"/>
```

Comment:
This caught my attention. When I import it into the old designer (.bpmn2 extension), the type is shown correctly. But when I open it in the new designer either on a web or in the Business Central, the type there is still `com.jboss.soap.service.acmedemo.FlightRequest`. Probably it doesn't matter as correct marshalling is chosen anyway, but can you make it the right type, i.e. `com.jboss.soap.service.acmedemo.ListAvailablePlanes`, so it doesn't confuse anybody in the future?

Reply:
good catch up, thanks for reviewing @MarianMacik